api-samples/sandbox/sandbox/sandbox.md

```diff
@@ -0,0 +1,10 @@
+# Sandbox
+
+This sample creates a tab with a sandboxed iframe (`sandbox.html`) to which the main page (`mainpage.html`)
+passes a counter variable. The sandboxed page uses the
+Handlebars template library to evaluate and compose a message
+using the counter variable which is then passed back to the main page for rendering.
+
+## Overview
+
+The default packaged app Content Security Policy (CSP) value disallows the use of `eval()` so using a sandbox is necessary to use external resources for this extension.
```

Comment:
I think this could be phrased better. Before I can suggest something, I need to know what a 'default packaged app' is.

Reply:
I think this should just say extension. I think this was written in the Chrome apps times.